src/internal_rep.rs

```diff
@@ -2267,6 +2406,9 @@ impl<'a> ValidatedStatement<'a> {
             ValidatedStatement::FcRule(f) => {
                 ValidatedStatement::FcRule(f.get_renamed_statement(renames))
             }
+            // Not 100% sure what to do here since we are dealing with everything as cascade strings
```

Comment:
I think the domtrans issue blocking this was fixed in f99eb814a50f4e4224f1054921739ad79cada455.  It might be that this still needs updating from the domtrans perspective.  But I think from the resourcetrans standpoint, it needs to implement a get_renamed_statement() function like the other variants.